hnn_core/viz.py

```diff
@@ -1185,3 +1198,234 @@ def plot_laminar_csd(times, data, contact_labels, ax=None, colorbar=True,
+
+                pos = self.net.pos_dict[cell_type][gid_idx]
+                pos = (float(pos[0]), float(pos[2]), float(pos[1]))
+
+                cell.plot_morphology(ax=self.ax, show=False,
+                                     pos=pos, xlim=self.xlim,
+                                     ylim=self.ylim, zlim=self.zlim)
+
+    def _update_axes(self):
+        self.ax.set_xlim(self._xlim)
+        self.ax.set_ylim(self._ylim)
+        self.ax.set_zlim(self._zlim)
+
+        self.ax.view_init(self._elev, self._azim)
+
+    # Axis limits
+    @property
+    def xlim(self):
+        return self._xlim
+
+    @xlim.setter
+    def xlim(self, xlim):
+        self._xlim = xlim
+        self.ax.set_xlim(self._xlim)
+
+    @property
+    def ylim(self):
+        return self._ylim
+
+    @ylim.setter
+    def ylim(self, ylim):
+        self._ylim = ylim
+        self.ax.set_ylim(self._ylim)
+
+    @property
+    def zlim(self):
+        return self._zlim
+
+    @zlim.setter
+    def zlim(self, zlim):
+        self._zlim = zlim
+        self.ax.set_zlim(self._zlim)
+
+    # Eleevation and azimuth of 3D viewpoint
+    @property
+    def elev(self):
+        return self._elev
+
+    @elev.setter
+    def elev(self, elev):
+        self._elev = elev
+        self.ax.view_init(self._elev, self._azim)
+
+    @property
+    def azim(self):
+        return self._azim
+
+    @azim.setter
+    def azim(self, azim):
+        self._azim = azim
+        self.ax.view_init(self._elev, self._azim)
+
+    # Minimum and maximum voltages
+    @property
+    def vmin(self):
+        return self._vmin
+
+    @vmin.setter
+    def vmin(self, vmin):
+        self._vmin = vmin
+        self.vsec_array = self.get_voltages()
+        self.color_array = self.colormap(self.vsec_array)
+
+    @property
+    def vmax(self):
+        return self._vmax
+
+    @vmax.setter
+    def vmax(self, vmax):
+        self._vmax = vmax
+        self.vsec_array = self.get_voltages()
+        self.color_array = self.colormap(self.vsec_array)
+
+    # Time and trial indices
+    @property
+    def trial_idx(self):
+        return self._trial_idx
+
+    @trial_idx.setter
+    def trial_idx(self, trial_idx):
+        self._trial_idx = trial_idx
+        self.vsec_array = self.get_voltages()
+        self.color_array = self.colormap(self.vsec_array)
+
+    @property
+    def time_idx(self):
+        return self._time_idx
+
+    @time_idx.setter
+    def time_idx(self, time_idx):
+        self._time_idx = time_idx
+        self.update_section_voltages(self._time_idx)
+
+    # Background color and voltage colormaps
+    @property
+    def bg_color(self):
+        return self._bg_color
+
+    @bg_color.setter
+    def bg_color(self, bg_color):
+        self._bg_color = bg_color
+        self.ax.set_facecolor(self._bg_color)
+
+    @property
+    def voltage_colormap(self):
+        return self._voltage_colormap
+
+    @voltage_colormap.setter
+    def voltage_colormap(self, voltage_colormap):
```

Comment:
an `export` method? I think matplotlib allows exporting animations ... I want to see HNN animations in presentations of people I don't know ;-)